management-api-agent-common/src/main/java/io/k8ssandra/metrics/http/NettyMetricsHttpServer.java

```diff
@@ -0,0 +1,39 @@
+package io.k8ssandra.metrics.http;
+
+import io.netty.bootstrap.ServerBootstrap;
+import io.netty.channel.EventLoopGroup;
+import io.netty.channel.epoll.EpollServerSocketChannel;
+import io.netty.handler.logging.LogLevel;
+import io.netty.handler.logging.LoggingHandler;
+import io.netty.handler.ssl.SslContext;
+import io.netty.handler.ssl.SslContextBuilder;
+import io.netty.handler.ssl.util.SelfSignedCertificate;
+
+import javax.net.ssl.SSLException;
+import java.security.cert.CertificateException;
+
+public class NettyMetricsHttpServer {
+
+    public void start(EventLoopGroup group) {
+        // Configure SSL.
+//        final SslContext sslCtx = buildSslContext();
+
+        ServerBootstrap b = new ServerBootstrap();
+        ServerBootstrap channel = b.group(group)
+                .childHandler(new NettyHttpInitializer(null))
+//                .handler(new LoggingHandler(LogLevel.DEBUG))
```

Comment:
We get Netty's logs indicating when channel is processed, such as ``READ READY [0x..]``. They're only useful for debugging the handler's process, but would make debugging everything else more difficult. Just enable it if you're developing the handler, otherwise - no.

Reply:
gotcha 👍